grid2op/Environment/BaseEnv.py

```diff
@@ -91,6 +91,8 @@ def __init__(self,
                  opponent_class=BaseOpponent,
                  opponent_budget_class=UnlimitedBudget,
                  opponent_init_budget=0.,
+                 opponent_attack_duration=12*4,
+                 opponent_attack_cooldown=12*24,
```

Comment:
same here: fall back to default behaviour in "make"

Reply:
Isn't the default `0` ?